### ISSUE TITLE
Copy  EEXIST: file already exists, mkdir

### ISSUE THREAD

**Issue by alan-agius4** [driftyco/ionic-app-scripts#347]
copy tasks gives error when copying to same folder (sometimes)

#### Short description of the problem:
When I provide the copy task to copy files to the same folder sometimes I get an error.
```
EEXIST: file already exists, mkdir
```

```

{
	src: "node_modules/ionicons/dist/fonts/",
	dest: "{{WWW}}/assets/fonts/"
},
{
	src: "{{SRC}}/assets/fonts/",
	dest: "{{WWW}}/assets/fonts/"
}
```

**Which @ionic/app-scripts version are you using?** 0.0.39

**Comment by danbucholtz** [comment]
Does the content still get copied over? It looks like it does on my end.

Thanks,
Dan

**Comment by alan-agius4** [comment]
Not only contents do get copied.

It looks like the dir get 'locked', until the first copy task is finished.
Seems like the issue is that the process is running in parallel and trying
to write to the same destination.

If i specify two different dest even if one of the directory is a sub of
the other it works fine.

**Comment by alan-agius4** [comment]
Not only contents do get copied.

It looks like the dir get 'locked', until the first copy task is finished.
Seems like the issue is that the process is running in parallel and trying
to write to the same destination.

If i specify two different dest even if one of the directory is a sub of
the other it works fine.

A bit of a hacky work around i found was to use glob


``` js
const glob = require("glob");

var pathsToCopy = [];
glob.sync("src/assets/**", {nodir: true}).foreach((value)=>{
  pathsToCopy.push({
      dest: value.replace("src", "{{SRC}}"),
      dest: value.replace("src", "{{WWW}}")
  });
});
```

than append this array to the copy `includes` files

**Comment by danbucholtz** [comment]
We are going to change this to a glob. Just need time to do it.

Thanks,
Dan

**Comment by danbucholtz** [comment]
Hi all,

This is resolved in `0.0.48` which was just published.

Please let me know if you have any issues.

Thanks,
Dan
